PixivImage.py

```diff
@@ -150,12 +151,15 @@ def __init__(self,
                 self.originalArtist = self.artist
 
             # parse image
-            self.ParseInfo(payload)
+            self.ParseInfo(payload,writeRawJSON)
```

Comment:
missing space here :P I'm using pyflake to lint the code, so it shows up as warning